io/io/src/TFile.cxx

```diff
@@ -4068,6 +4068,13 @@ TFile *TFile::OpenFromCache(const char *name, Option_t *, const char *ftitle,
 /// In RECREATE mode, a nullptr is returned if the file can not be created.
 /// In UPDATE mode, a nullptr is returned if the file cannot be created or opened.
 
+std::unique_ptr<TFile> OpenU(const char *url, Option_t *options, const char *ftitle,
+                   Int_t compress, Int_t netopt)
+{
+   std::unique_ptr<TFile> uniquePtr = std::make_unique<TFile>(&url, &options, &ftitle, compress, netopt);
```

Comment:
```suggestion
   std::unique_ptr<TFile> uniquePtr{ TFile::Open(&url, &options, &ftitle, compress, netopt) };
```
Not that `make_unique` will call the constructor of the (argument) class (i.e. `TFile`) and this is different behavior that `TFile::Open`.  `TFile::Open` is factory function that depending on its argument (filename pointing to a local file or to remote file) will constructor an object a different type (eg. `TFIle` itself for local file or `TNetFile` for some remote files and others types)>

Reply:
Ahh I see, will make the change accordingly 👍